plugins/pi4j-plugin-ffm/src/main/java/com/pi4j/plugin/ffm/providers/spi/SpiFFM.java

```diff
@@ -127,4 +133,21 @@ private void checkClosed() {
             throw new Pi4JException("SPI bus  '" + path + "' is closed");
         }
     }
+
+    public static void main(String[] args) throws IOException, InterruptedException {
```

Comment:
Why do we have a main class in production code? This should probably go to a test, no?